--- a/apps/openmw/mwworld/actionteleport.cpp
+++ b/apps/openmw/mwworld/actionteleport.cpp
@@ -83,9 +83,7 @@
                 if (hostilePursuer)
                 {
                     MWBase::World* world = MWBase::Environment::get().getWorld();
-                    const ESM::CellId fromCellId = it->getCell()->getCell()->getCellId();
-                    const std::string fromCellName = it->getCell()->isExterior()
-                        ? std::string() : it->getCell()->getCell()->mName;
+                    const ESM::Cell& fromCell = *it->getCell()->getCell();
                     const ESM::Position fromPosition = it->getRefData().getPosition();
 
                     MWWorld::CellStore* destinationCell = nullptr;
@@ -101,8 +99,8 @@
 
                     if (destinationCell)
                     {
-                        sequence.recordDoorTransition(fromCellId, fromCellName, fromPosition,
-                            destinationCell->getCell()->getCellId(), mPosition);
+                        sequence.recordDoorTransition(fromCell, fromPosition,
+                            *destinationCell->getCell(), mPosition);
                     }
                 }
 
