--- /mnt/data/x042_final/apps/openmw/mwmp/processors/player/ProcessorGUIMessageBox.hpp	2026-08-30 06:14:12.000000000 +0000
+++ /mnt/data/x042a_final/apps/openmw/mwmp/processors/player/ProcessorGUIMessageBox.hpp	2026-08-30 07:02:36.997818847 +0000
@@ -9,6 +9,7 @@
 #include "../../../mwbase/environment.hpp"
 #include "../../../mwbase/windowmanager.hpp"
 #include "../../../mwgui/dialogue.hpp"
+#include "../../../mwworld/cellstore.hpp"
 #include <components/misc/stringops.hpp>
 
 #include <utility>
--- /mnt/data/x042_final/apps/launcher/graphicspage.cpp	2026-08-30 06:09:18.000000000 +0000
+++ /mnt/data/x042a_final/apps/launcher/graphicspage.cpp	2026-08-30 07:02:36.998048957 +0000
@@ -1203,7 +1203,7 @@
     Settings::Manager::setBool("idle actor ripples", "Water", level >= 2);
     Settings::Manager::setInt("max shader ripples", "Water", level == 0 ? 1 : (level == 1 ? 2 : (level <= 3 ? 4 : (level == 4 ? 6 : 8))));
     Settings::Manager::setFloat("small feature culling pixel size", "Water",
-        level == 0 ? 32.f : (level == 1 ? 28.f : (level == 2 ? 20.f : (level == 3 ? 18.f : (level == 4 ? 14.f : 10.f))));
+        level == 0 ? 32.f : (level == 1 ? 28.f : (level == 2 ? 20.f : (level == 3 ? 18.f : (level == 4 ? 14.f : 10.f)))));
     Settings::Manager::setFloat("refraction scale", "Water",
         level <= 1 ? 4.f : (level <= 3 ? 3.f : (level == 4 ? 2.f : 1.f)));
 
